templates/starter/bicep/repo.yaml

```diff
@@ -42,3 +42,6 @@ repo:
     - from: ../../common/.devcontainer/devcontainer.json/bicep-starter/devcontainer.json
       to: ./.devcontainer/devcontainer.json
 
+    # core bicep files
+    - from: ../../common/infra/bicep/core
```

Comment:
I would also like to include abbreviations.json /infra/bicep/core

Reply:
It's already included on line 34

Reply:
Confirmed that the generated repo has it, as well.